learningequality/studio
# New command palette arrangement, and responsive design

## Summary
### Description of the change(s) you made
Rearrange command palette and introduce the KListWithOverflow component to add responsive design.

### Screenshots (if applicable)

https://github.com/learningequality/studio/assets/51239030/5f7c267a-6904-4839-b4b9-27fbacabce93



___
## Reviewer guidance
### How can a reviewer test these changes?
1. Go and edit the nodes of a channel
2. Select multiple topics and resources
3. Resize your screen and see that everything is displaying ok

## References
Closes #3423.

Added [@vue/composition-api](https://github.com/learningequality/studio/pull/4446/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R59)

## Comments
Vue composition api was added to support the new KDS component and to allow the usage of composition api in studio in the future. I have tested studio on several places and didnt find anything wrong with it.

----

### Contributor's Checklist


PR process:


Studio-specifc:



Testing:

___

### Reviewer's Checklist
#### This section is for reviewers to fill out.
